account/models.py

```diff
@@ -5,80 +5,80 @@
 
```

Comment:
My autoformatter does 4 spaces for tab. Is there a clean way for us to ignore/resolve these formatting diffs?